roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -293,7 +293,7 @@ docker_plugins: []
 etcd_kubeadm_enabled: false
 
 # Containerd options
-containerd_use_systemd_cgroup: false
+containerd_use_systemd_cgroup: true
```

Comment:
you removed this var from config templates and do not use anywhere...

Reply:
it's used in this file `roles\kubernetes\node\tasks\facts.yml` to set the kubelet cgroup driver.

Reply:
containerd 1.4.x work only with systemd driver ?

Reply:
As the default containerd 1.4 runtime is now `io.containerd.runc.v2` which supports cgroups v2, the option to set the cgroup to either `cgroupfs` or `systemd` is effectively redundant. 

https://github.com/containerd/containerd/blob/master/pkg/cri/config/config.go#L213-L216

Hence the Kubelet can be set as default to `systemd` as recommended in the Kubernetes docs.

https://kubernetes.io/docs/setup/production-environment/container-runtimes/#cgroup-drivers

Reply:
Most distro are still using cgroup v1, so not sure this is redundant at all, but using systemd is recommended for some time now. We could stop forcing croup v1 for Fedora.